docs/getting_started/integrating_into_django.md

```diff
@@ -118,4 +118,4 @@ Superuser accounts receive automatic access to the Wagtail admin interface; use
 
 You're now ready to add a new app to your Django project (via `./manage.py startapp` - remember to add it to `INSTALLED_APPS`) and set up page models, as described in [Your first Wagtail site](/getting_started/tutorial).
 
-Note that there's one small difference when not using the Wagtail project template: Wagtail creates an initial homepage of the basic type `Page`, which does not include any content fields beyond the title. You'll probably want to replace this with your own `HomePage` class - when you do so, ensure that you set up a site record (under Settings / Sites in the Wagtail admin) to point to the new homepage.
\ No newline at end of file
+Note that there's one small difference when not using the Wagtail project template: Wagtail creates an initial homepage of the basic type `Page`, which does not include any content fields beyond the title. You'll probably want to replace this with your own `HomePage` class - when you do so, ensure that you set up a site record (under Settings / Sites in the Wagtail admin) to point to the new homepage.
```

Comment:
note: this is some kind of whitespace change but for the life of me can't spot what it is

Reply:
Apparently there was no newline at the end of the file in the old version. (I used the Github online editor to make this change, so that must add it automatically...)

Reply:
I could swear the diff view in GitHub used to show the lack of EOF new lines. Oh well 😂